network/src/commonMain/kotlin/com/wire/kalium/network/BackendMetaDataUtil.kt

```diff
@@ -24,8 +24,8 @@ import com.wire.kalium.network.api.base.unbound.versioning.VersionInfoDTO
 import com.wire.kalium.network.tools.ApiVersionDTO
 import com.wire.kalium.network.tools.ServerConfigDTO
 
-val SupportedApiVersions = setOf(0, 1, 2, 3, 4)
-val DevelopmentApiVersions = setOf(5)
+val SupportedApiVersions = setOf(0, 1, 2, 3, 4, 5)
+val DevelopmentApiVersions = setOf(6)
```

Comment:
can we not add api v6 to development yet, lets keep it to only 5 since this is the version 4.6 will be released with + it will break dev builds if the server introduced changes?

Reply:
I think there is a safe point in this, so for the develop cherry-pick we make sure to enable it anyway

Reply:
okay, but I'll keep all the new files, so later it will be easier to update :)